### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ landing page for database devops
* Jira/GitHub Issue numbers (if any): \ DBOPS-596
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -686,6 +673,55 @@ const sidebars = {
...
+      },
+      collapsed: true,
+      items: [{ type: "autogenerated", dirName: "database-devops" }],
+    }, 
+    // Release Notes
+    {
+      type: "link",
+      label: "Release Notes",
+      className: "sidebar-Release_Notes",
+      href: "/release-notes/database-devops",
```
Comment (ravilach) [inline sidebars.js:699]: Does this page exist?

Comment (jenniferopal) [reply sidebars.js:699]: Yes, I deployed it today. 
https://developer.harness.io/release-notes/database-devops

File: sidebars.js [hunk 2]
```diff
@@ -686,6 +673,55 @@ const sidebars = {
...
     },
   ],
+  
+  databasedevops: [
+    // Database DevOps Landing Page
+    {
+      type: "category",
+      label: "Database DevOps",
+      className: "sidebar-dbdevops",
+      // link: {
```
Comment (wei-harness) [inline sidebars.js:683]: If you have a custom landing page, you may try to uncomment those 3 lines marked in the attached screenshot,
also you have to create a doc with id "database-devops" in the under docs,
e.g.: for CD, you can find the doc with id continuous-delivery here: /docs/continuous-delivery.md, and import the custom Component you created for DB Devops.

Comment (jenniferopal) [reply sidebars.js]: Thanks so much for this. This made such a difference to all the errors I was getting.

**Review by HarnessTheChaos — APPROVED** [review]
Reviewed new DB DevOps docs for landing page based on the announcement of the new module on 9/25. LGTM.

**Review by wei-harness — APPROVED** [review]
LGTM

**Pull request merged**